getsession/internal/config/config.go

```diff
@@ -0,0 +1,47 @@
+package config
+
+import (
+	"log"
+	"os"
+	"strconv"
+
+	"github.com/joho/godotenv"
+)
+
+type Config struct {
+	TelegramApiId   int
+	TelegramApiHash string
+	Phone           string
+}
+
+func LoadConfig() Config {
+	LoadEnvFile()
+	config := InitConfig()
+	CheckConfigFields(config)
+	return config
+}
+
+func CheckConfigFields(config Config) {
+	if config.TelegramApiId == 0 ||
+		config.TelegramApiHash == "" ||
+		config.Phone == "" {
+		log.Fatalf("One or more environment variables are missing. Check .env file.")
+	}
+}
+
+func InitConfig() Config {
+	telegramApiId, _ := strconv.Atoi(os.Getenv("TELEGRAM_API_ID"))
+	config := Config{
+		TelegramApiId:   telegramApiId,
+		TelegramApiHash: os.Getenv("TELEGRAM_API_HASH"),
+		Phone:           os.Getenv("PHONE"),
+	}
+	return config
+}
+
+func LoadEnvFile() {
+	err := godotenv.Load()
```

Comment:
loading from .env is convenient for local development, but in the case of deployment it strongly requires for now existing file `.env`, at least empty otherwise there will be log.Fatal. It is not fatal situation for the program, it is an option to configure the program. Let's make it really optional without error messages.